test/models/test_openai_model.py

```diff
@@ -23,9 +23,12 @@
 
```

Comment:
How about adding a test to verify whether the actual backend model and context length matches what we set?

Reply:
> How about adding a test to verify whether the actual backend model and context length matches what we set?

done